### DISCUSSION TITLE
Redirection from "example.com/path/" to "example.com/path" with 308 Permanent Redirect status code

### DISCUSSION THREAD

**Discussion by adenlall** [vercel/next.js#62508] [Help] [answered]
### Summary

How to prevent redirecting from any path that contain `/` at the end to the same path without it, with 308 status code then 200 status code like this : 

example : 
redirect : `https://nextjs.org/docs/` **308**
to :            `https://nextjs.org/docs` **200**

<a href="https://ibb.co/S0HmRhX"><img src="https://i.ibb.co/ftZN4hY/Screenshot-2024-02-25-at-15-18-00-Bulk-URL-HTTP-Status-Code-Header-Redirect-Checker.png" alt="Screenshot-2024-02-25-at-15-18-00-Bulk-URL-HTTP-Status-Code-Header-Redirect-Checker" border="0"></a>

and it happen in all routs (dynamic, statics...) 

### Additional information

```js
my dependencies :
    "@vercel/analytics": "^1.1.3",
    "cheerio": "^1.0.0-rc.12",
    "html-react-parser": "^5.1.1",
    "next": "14.1.0",
    "nextjs-progressloader": "^1.1.0",
    "react": "^18",
    "react-dom": "^18",
    "swiper": "^11.0.5"
nodejs version : 21
```

**Comment by icyJoseph** [comment, marked as answer]
Hi,

This a bit of an annoying thing, but trailing slash `/` is handled as either yes, or not, in Next.js apps. You can read more about it here: 

- https://nextjs.org/docs/app/api-reference/next-config-js/trailingSlash
